module2/homework/transformContainers/transform.cpp

```diff
@@ -0,0 +1,33 @@
+#include "transform.hpp"
+
+#include <algorithm>
+
+bool anyContainerIsEmpty(const strList& strList, const intDeque& intDeq) {
+    return strList.empty() || intDeq.empty();
+}
+
+bool containersSizeIsNotEqual(const strList& strList, const intDeque& intDeq) {
+    return strList.size() != intDeq.size();
+}
+
+intStrMap removeDuplicateAndTranformToMap(strList strList, intDeque intDeq) {
+    intStrMap returnValue{};
+    if(anyContainerIsEmpty(strList, intDeq)) { 
+        return returnValue;
+    }
```

Comment:
What's point of creating function for comparing(considering that you are perfoming this type of compare only once). You could just do comparing here.

Reply:
I'd created it for better code clarity/ readibility. But you're argument that I'd used it only once is very strong. I'll change it.